openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -181,6 +254,13 @@ def test_patch_account_different_user(self):
         self.different_client.login(username=self.different_user.username, password=TEST_PASSWORD)
         self.send_patch(self.different_client, {}, expected_status=404)
 
+    def test_patch_account_is_staff(self):
```

Comment:
And we may choose to promote the anonymous test case up into the base class (which is one of the refactorings I had done on your branch).